inc/altis-config/class-qm-collector-altis-config.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Altis Config data collector for the Query Monitor plugin.
+ */
+
+namespace Altis\Dev_Tools;
```

Comment:
As discussed on slack this namespace will need to match the directory structure eg. `Altis\Dev_Tools\Altis_Config`. I know I mentioned renaming the directory to just `config` but on reflection it's good to be explicit about what this is. Could be confused as "configuration for the dev tools module" with my suggestion.